lib/grinder_files.dart

```diff
@@ -336,13 +339,12 @@ Directory getDir(String path) {
 }
 
 void copy(FileSystemEntity entity, Directory destDir,
-    [GrinderContext context]) {
+    [GrinderContext? context]) {
```

Comment:
It looks like the `context` arg was never used in `_copyImpl()`. We should mark this param as deprecated.